### ISSUE TITLE
Add iframe title to template 3 iframe (rufous-sandbox) and have a lang attribute

### ISSUE THREAD

**Issue by leahbannon** [18F/federalist#404]

**Comment by wslack** [comment]
@leahbannon any idea what this means? I'm mystified.

**Comment by atav32** [comment]
https://stackoverflow.com/questions/12133617/strange-iframe-added-into-body-tags-called-rufous-sandbox

**Comment by jseppi** [comment]
What Federalist template is this referring to?

**Comment by atav32** [comment]
It might not even be referring to a template? According to the StackOverflow answer, `rufous-sandbox` is added by a Twitter script.

(Also I have no idea what this repo is, I just found it when Googling "rufous-sandbox" 😆)

**Comment by jseppi** [comment]
Ha, ok, thanks for the context @atav32!

**Comment by el-mapache** [comment]
Based on the context, I think this can be closed @wslack. I looked at all the sample templates, and none of them seems to load an iframe or any twitter code.
